docs/src/pages/[platform]/connected-components/authenticator/headless/index.page.mdx

```diff
@@ -18,6 +18,6 @@ export async function getStaticProps() {
   return { props: {} }
 }
 
-<Fragment useCommonWebContent platforms={['web', 'flutter']}>
+<Fragment useCommonWebContent platforms={['web', 'react-native']}>
```

Comment:
Instead of having web and react-native as completely separate pages, we could bring the `headless-usage.web.mdx` contents up to this file so that we don't have a mostly duplicated file. Thoughts?

Reply:
Done